tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,80 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Box, Input, Button, Section } from 'tgui/components';
+import { ColorBox, Flex } from '../components';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const peripherals = data.peripherals || [];
+  const { textInput } = props;
```

Comment:
How is this prop flowing into the component? Interfaces typically don't use props.